R/1_nflverse.R

```diff
@@ -48,6 +48,11 @@
         "player_name"="full_name","pos"="position","team"
       ))
     ) %>%
+    dplyr::group_by(season, gsis_id, sportradar_id) %>%
```

Comment:
add .data prefix to avoid rcmdcheck issues

```suggestion
    dplyr::group_by(.data$season, .data$gsis_id, .data$sportradar_id) %>%
```